builders/get-started/networks/moonbeam-dev.md

```diff
@@ -10,190 +10,189 @@ description: Follow this tutorial to learn how to set up your first Moonbeam nod
 ```
 
 !!! note
-    For people not familiar with Substrate, the `--dev` flag is a way to run a Substrate-based node in a single node developer configuration for testing purposes. You can learn more about `--dev` in [this Substrate tutorial](https://substrate.dev/docs/en/tutorials/create-your-first-substrate-chain/interact){target=_blank}.
+    For people not familiar with Substrate, the `--dev` flag is a way to run a Substrate-based node in a single-node developer configuration for testing purposes. When you run your node with the `--dev` flag, your node is started in a fresh state, and its state is not persisted.
 
 You should see an output that looks like the following, showing an idle state waiting for blocks to be produced:
 
 ![Output shows blocks being produced](/images/builders/get-started/networks/moonbeam-dev/moonbeam-dev-4.png)
 
-For more information on some of the flags and options used in the example, check out [Flags and Options](#commands-flags-and-options). If you want to see a complete list of all of the flags, options, and subcommands, open the help menu by running:
+For more information on some of the flags and options used in the example, check out the [Flags](#node-flags) and [Options](#node-options). If you want to see a complete list of all of the flags, options, and subcommands, open the help menu by running:
 
 ```
 ./target/release/moonbeam --help
 ```
-## Connecting Polkadot.js Apps to a Local Moonbeam Node {: #connecting-polkadot-js-apps-to-a-local-moonbeam-node } 
-
-The development node is a Substrate-based node, so you can interact with it using standard Substrate tools. The RPC endpoint for HTTP and WS is:
-
- - **HTTP** - `{{ networks.development.rpc_url }}`
- - **WS** - `{{ networks.development.wss_url }}` 
-
-Start by connecting to it with Polkadot.js Apps. Open a browser to: [https://polkadot.js.org/apps/#/explorer](https://polkadot.js.org/apps/#/explorer){target=_blank}. This will open Polkadot.js Apps, which automatically connects to Polkadot MainNet.
-
-![Polkadot.js Apps](/images/builders/get-started/networks/moonbeam-dev/moonbeam-dev-5.png)
 
-Click on the top left corner to open the menu to configure the networks, then take the following steps:
+## Configure your Moonbeam Development Node {: #configure-moonbeam-dev-node }
 
-1. Navigate down to open the **Development** sub-menu
-2. Click on **Local Node**, which points Polkadot.js Apps to `ws://127.0.0.1:9944`. If not, you can enter it under **custom endpoint**
-3. Select the **Switch** button, and the site should connect to your Moonbeam development node
+Now that you know how to get a standard Moonbeam development node up and running, you may be wondering how you can configure it. The following sections will cover some common configurations you can use when you spin up your node.
 
-![Select local node](/images/builders/get-started/networks/moonbeam-dev/moonbeam-dev-6.png)
+### Configure Block Production {: #configure-block-production }
```

Comment:
I do like a section specific to this, but I feel the order is not right.

This is a flag, but then we have a section on `Common Flags to Configure your Node` and then `Common Options to Configure your Node` which also has a section on `--sealing`